### PR TITLE
Invalidate cloudfront after deployment

### PR DESCRIPTION
**Pull request by ben181231**
connect #49

### CODE REVIEW

File: tools/invalidate-cf.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+.forEach(function (perEnv) {
+  if (!process.env[perEnv]) {
+    console.error(`Error: \$${perEnv} is not set`);
+    process.exit(1);
+  }
+});
+
+const invalidatingPaths = [
+  '/app.js',
+  '/app.node.js',
```
Comment (rickmak) [inline tools/invalidate-cf.js]: As far as I know the app.node.js won't load in browser?

File: .travis.yml [hunk 2]
```diff
@@ -40,10 +36,12 @@ deploy:
...
+after_deploy:
+  - ./scripts/after-deploy.sh
 
 env:
   global:
     - PRODUCTION_BRANCH="production"
+    - secure: "u/3cyaOW8G4C3/oMSrWPNDEU6CcCmhPSweDwc1EEf/ch0jVC4LfQxtE9kLQRNO9uxCUOgZohMvBn+pArw7b6VPSjnMUhB6rldfjK7R2k+tLH/eSnkCQYaETLoZDqquJmzfrYcFd++pdSwZ8hFRgAOG7UkbII+cK88Q7ZYVeP6HwElLTm+zmyk7S56PQoohGVRgfJbSK34hLJfXMAG7n1nk+WckRX0CmmTMyZlrNOj6XeehXiZLhxY3/uKz0LvMv6ZLK3TgPtwz6jDrUi3uDn1Be6q4DrD3nkcmxjIRrmtvaxYYDw/Ulheg+eOuQwfMvr2RiUgeYOMDCqgqO2ljoMEvcZEBEc41WqHFakDHKWBOA4ideicVY1Y6QWpUah2p/ATPaPi3j0JkMtm3eer60+587viGfyomiH7YoeO5hUAXopipd0cmy+VzEzE7Jvw7vfes4isG+g7JWvKLh6L97tX9YhtmLcffqAmb8p7jJzn9kFMdRAYd+zsAgf0miQDUtDnhT/BTxHHafYvrTXOL+iAm9AVazyLr1XqsPtY2NxhhIK46ePEB6ivpqCaKnrVcvSVdxPTRvcQgnmz36MF4/OXLVz95x8kUOzrK0a8+OP8w7+M+iGcic0zZdMqG7iFKv6zRthx67wGgg71VzmPaqhHa6fxHAj6w7VEK68NJ/6fjg="
```
Comment (rickmak) [inline .travis.yml]: Add comment to which is which..... Or commit in separate commit.

File: tools/invalidate-cf.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  invalidatingPaths.length < 1000 ? invalidatingPaths.length : 1000;
+
+const v = new Void({
+  paths : invalidatingPaths,
+  maxPaths
+});
+
+setInterval(function () {
+  let hasWaitingJob = false;
+  v.queue.forEach(function (perJob) {
```
Comment (rickmak) [inline tools/invalidate-cf.js]: Void provide callback. Should use callback instead of pull here.

Comment (ben181231) [inline tools/invalidate-cf.js]: But the callback will wait for the invalidation finish.

File: tools/invalidate-cf.js (hunk 1 above)
Comment (ben181231) [inline tools/invalidate-cf.js]: Oh. Yes.

File: .travis.yml (hunk 2 above)
Comment (ben181231) [inline .travis.yml]: Let me add comment to that.

File: tools/invalidate-cf.js (hunk 3 above)
Comment (rickmak) [inline tools/invalidate-cf.js]: Oh , i see why.

**Pull request merged**